docs/guide/writing/style-guides.md

```diff
@@ -0,0 +1,91 @@
+
+If you happen to have a background in academia or journalism, you will probably be familiar the [AP stylebook](https://en.wikipedia.org/wiki/AP_Stylebook) or the [Chicago manual of style](http://www.chicagomanualofstyle.org/book/ed17/frontmatter/toc.html). Those are great resources for writing in general, particularly for grammar and syntax, but if you're reading this page chances are you are considering style in the context of technical documentation.
+
+### Technical Document
+Style guides help you write a variety of content, such as documenting the methods of an API, or presenting a overview of complex technical concepts, or focusing on how to write particular content like user manuals, release notes, or tutorials.
+
+Other style guides often focus on the reader, helping you communicate to developers, product managers, the general public, and others. They can also focus on the language itself (tone, style, grammar). Others provide best practices on how to manage your content, version control, or publication and delivery strategies.
+
+## Why do I need a style guide?
+A practical reason to use a style guide is that they help you write content. Human languages are extremely flexible, and there are many ways in which a particular message can be communicated. By following a style guide you limit the variation, making it easier for you to focus on getting your message across. This makes style guides extremely useful for people joining projects.
+
+Style guides also increase consistency in your content. There are good reasons why you want to keep a consistent tone, voice and style in your documentation. Marketing-oriented folks understand the importance of voice and tone in building a brand identity, a strategy we can extend into technical writing. Consistency also has a big impact on how effectively you communicate -- that is, on how well you manage to transfer a particular information to your audience.
+
+There's science behind this. Our brains appear to be hardwired to identify differences -- anything that stands out from its context will catch our attention. A lot of variation will drain our cognitive resources, making it more difficult for us to assimilate information. The degree to which cognitive resources are drained varies between people, but in general it is a good idea to minimize the [cognitive load](https://en.wikipedia.org/wiki/Cognitive_load) of your communications.
+
+## Different styles for different folk
+
+Choosing the right style guide for your project depends on your particular context:
+  - Who you are
+  - Who you are writing for
+  - What you are writing about
+
+> If you really need to pick a style guide ***right now***, we recommend you take a look at the [Google Developer Documentation Style Guide](https://developers.google.com/style/)
```

Comment:
I'm not sure this is the one we want to recommend as tl;dr

Reply:
Agreed

Reply:
Agreed. Let's not make a recommendation here.

Reply:
removed